plugins/mobile-plugin/src/main/java/ru/sbtqa/tag/pagefactory/mobile/support/AdbConsole.java

```diff
@@ -21,7 +21,8 @@ public static boolean execute(String command) {
     }
 
     public static boolean execute(String deviceUDID, String command) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "deviceUDID". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)